### PR TITLE
Membership login

### PR DESCRIPTION
**Pull request by skovsboll**
This was made to test https://github.com/Ucommercenet/Ucommerce/pull/334

Adding a login form was the easiest way to test membership, but the form is useful as an example of membership in Umbraco and Ucommerce.

Skærmbillede 2020-06-04 10 55 12

### CODE REVIEW

File: src/AvenueClothing/Models/LoginModel.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace AvenueClothing.Models
+{
+    public class LoginViewModel
+    {
+        public string Login { get; set; }
+
+        public string Password { get; set; }
+
```
Comment (ilny) [inline src/AvenueClothing/Models/LoginModel.cs:10]: I don't see a reason for the empty lines.

File: src/AvenueClothing/Views/Membership/LoginForm.cshtml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+﻿@using AvenueClothing.Controllers
+@using Umbraco.Web
+@model AvenueClothing.Models.LoginViewModel
+
+@if (User.Identity.IsAuthenticated)
+{
+    <p>
+        Welcome, @User.Identity.Name
+    </p>
```
Comment (ilny) [inline src/AvenueClothing/Views/Membership/LoginForm.cshtml:9]: Why not leave it as 1 line?
The content is simple.

File: src/AvenueClothing/css/uCommerce.demostore.css [hunk 3]
```diff
@@ -642,4 +642,12 @@ p.btn-group.view-details {
     text-decoration: line-through;
     margin-bottom: 5px;
         opacity: 0.5;
+}
+
+.editor-label {
+    margin-top: 10px;
+}
+
+input[type=submit].btn {
```
Comment (ilny) [inline src/AvenueClothing/css/uCommerce.demostore.css:651]: Very generic selector!

File: src/AvenueClothing/Views/Membership/LoginForm.cshtml (hunk 2 above)
Comment (skovsboll) [reply src/AvenueClothing/Views/Membership/LoginForm.cshtml:9]: The formatter inside Rider suggests this style. It's not less simple this way IMO.

File: src/AvenueClothing/Models/LoginModel.cs (hunk 1 above)
Comment (skovsboll) [reply src/AvenueClothing/Models/LoginModel.cs:10]: Removed the empty lines.

File: src/AvenueClothing/css/uCommerce.demostore.css (hunk 3 above)
Comment (skovsboll) [reply src/AvenueClothing/css/uCommerce.demostore.css]: Improved it.

**Pull request merged**